app/src/main/res/layout/activity_link_device.xml

```diff
@@ -12,34 +12,36 @@
         android:layout_height="match_parent">
 
         <com.google.android.material.tabs.TabLayout
-            android:id="@+id/tabLayout"
             style="@style/Widget.Session.TabLayout"
-            android:layout_width="match_parent"
-            android:layout_height="@dimen/tab_bar_height" />
+            android:id="@+id/tabLayout"
 
-        <FrameLayout
             android:layout_width="match_parent"
-            android:layout_height="match_parent"
-            android:animateLayoutChanges="true">
+            android:layout_height="@dimen/tab_bar_height" />
 
-            <RelativeLayout
-                android:id="@+id/loader"
+        </androidx.viewpager.widget.ViewPager>
+            <FrameLayout
+                android:animateLayoutChanges="true"
                 android:layout_width="match_parent"
                 android:layout_height="match_parent"
-                android:background="#A4000000"
-                android:visibility="gone">
 
-                <com.github.ybq.android.spinkit.SpinKitView
-                    style="@style/SpinKitView.Large.ThreeBounce"
-                    android:layout_width="wrap_content"
-                    android:layout_height="wrap_content"
-                    android:layout_centerInParent="true"
-                    android:layout_marginTop="8dp"
-                    app:SpinKit_Color="@android:color/white" />
+                <RelativeLayout
+                    android:id="@+id/loader"
+                    android:layout_width="match_parent"
+                    android:layout_height="match_parent"
+                    android:background="#A4000000"
+                    android:visibility="gone">
+
+                    <com.github.ybq.android.spinkit.SpinKitView
+                        style="@style/SpinKitView.Large.ThreeBounce"
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content"
+                        android:layout_marginTop="8dp"
+                        android:layout_centerInParent="true"
+                        app:SpinKit_Color="@android:color/white" />
 
-            </RelativeLayout>
+                </RelativeLayout>
 
-        </FrameLayout>
+            </FrameLayout>
 
     </androidx.viewpager.widget.ViewPager>
```

Comment:
This line will also need to be removed (the line wasn't originally removed it was just shifted down for some reason), there should be a build errors for this at the moment